packages/deployment/app/cloudformation.template.yml

```diff
@@ -1201,9 +1200,83 @@ Resources:
       FilterPattern: ""
 {{/each}}
 {{/if}}
+
+# ECS EC2 Autoscaling Policies
+{{# if ecs.autoscaling_policies}}
+  {{#each ecs.autoscaling_policies}}
+  {{@key}}ScalingPolicy:
+    Type: AWS::AutoScaling::ScalingPolicy
+    Properties:
+      AdjustmentType: {{ adjustment_type }}
+      AutoScalingGroupName:
+        Ref: CumulusECSAutoScalingGroup
+      {{#ifNotEquals policy_type "StepScaling"}}
+      Cooldown: {{# if cool_down }}{{ cool_down }}{{else}}60{{/if}}
+      {{/ifNotEquals}}
+      {{#ifNotEquals policy_type "SimpleScaling"}}
+      MetricAggregationType: {{# if metric_aggregation_type }}{{ metric_aggregation_type }}{{else}}Average{{/if}}
+      EstimatedInstanceWarmup: {{# if instance_warmup }}{{ instance_warmup }}{{else}}60{{/if}}
+      {{/ifNotEquals}}
+      {{#ifEquals adjustment_type "PercentChangeInCapacity" }}
+      MinAdjustmentMagnitude: {{# if min_adjustment }}{{ min_adjustment }}{{else}}1{{/if}}
+      {{/ifEquals}}
+      PolicyType: {{ policy_type }}
+      {{#ifEquals policy_type "SimpleScaling"}}
+      ScalingAdjustment: {{ scaling_adjustment }}
+      {{/ifEquals}}
+      {{#ifEquals policy_type "StepScaling"}}
```

Comment:
Do we want to add support for these without testing? I don't know the answer here

Reply:
Added unit tests